test_runner/performance/test_branch_creation.py

```diff
@@ -47,38 +47,39 @@ def test_branch_creation_heavy_write(neon_compare: NeonCompare, n_branches: int)
              'pitr_interval': '5 s'
          })
 
-    def run_pgbench(branch: str):
```

Comment:
See https://github.com/neondatabase/neon/pull/2143#discussion_r930386560 for the reason behind this change